### PR TITLE
Support json file import

### PR DESCRIPTION
**Pull request by maooson**
Use Case: Import content from JSON
`
.\sq.exe content import posts posts.json --fields=title,author,paragraphs --format=JSON
`

### CODE REVIEW

File: cli/Squidex.CLI/Squidex.CLI.Tests/MapJsonFileToSquidexTests.cs [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+{
+    public class MapJsonFileToSquidexTests
+    {
+        private readonly JsonTextReader jsonReader;
+
+        public MapJsonFileToSquidexTests()
+        {
+            var json = new StringBuilder();
+
+            using (var stringWriter = new StringWriter(json))
```
Comment (SebastianStehle) [inline cli/Squidex.CLI/Squidex.CLI.Tests/MapJsonFileToSquidexTests.cs]: Can you create an anonymous object here and serialize that to json.

Like 

```
var json = JsonConvert.SerializeObject(new {
   text = "..."
});

it is hard to understand what you create here.

**Comment by SebastianStehle** [comment]
Looks awesome :)

File: cli/Squidex.CLI/Squidex.CLI.Tests/MapJsonFileToSquidexTests.cs (hunk 1 above)
Comment (maooson) [reply cli/Squidex.CLI/Squidex.CLI.Tests/MapJsonFileToSquidexTests.cs]: Good idea, I will change it


> Can you create an anonymous object here and serialize that to json.
> 
> Like
> 
> ```
> var json = JsonConvert.SerializeObject(new {
>    text = "..."
> });
> 
> it is hard to understand what you create here.
> ```

**Comment by SebastianStehle** [comment]
Great, i want to do something today on the CLI as well and then I will release it in a few hours.

**Comment by SebastianStehle** [comment]
I have published it. In case you want to add something to the docs: Please create a PR: https://github.com/Squidex/squidex-docs2/blob/master/02-documentation/developer-guides/automation-tools.md#import-content-from-json

**Pull request merged**